Dockerfile.arm64

```diff
@@ -15,4 +15,5 @@ FROM arm64v8/alpine:3.7
 LABEL maintainer "Casey Davenport <casey@tigera.io>"
 
 ADD bin/kube-controllers-linux-amd64 /usr/bin/kube-controllers
```

Comment:
should this not be `arm64` rather than `amd64`?

Reply:
Yep, it probably should :)